### ISSUE TITLE
Code questions about training generators

### ISSUE THREAD

**Issue by ComeFromTheMars** [USTC-StarTeam/DR4SR#8]
Why add a reg_loss when training in Pretrain_regenerator?

**Comment by ShadowTinker** [comment]
Hi, @ComeFromTheMars ! 
This is an engineering effort aimed at balancing the learning across the K memory spaces. However, through some hyperparameter tuning, I discovered that this loss has minimal impact on the final outcome.

**Comment by ComeFromTheMars** [comment]
thanks

> Hi, @ComeFromTheMars ! This is an engineering effort aimed at balancing the learning across the K memory spaces. However, through some hyperparameter tuning, I discovered that this loss has minimal impact on the final outcome.
